infra/base-images/base-runner-debug/Dockerfile

```diff
@@ -15,4 +15,8 @@
 ################################################################################
 
 FROM gcr.io/oss-fuzz-base/base-runner
-RUN apt-get update && apt-get install -y gdb valgrind zip
+RUN apt-get update && apt-get install -y build-essential libgmp-dev valgrind zip
+
+RUN wget https://ftp.gnu.org/gnu/gdb/gdb-12.1.tar.xz && \
```

Comment:
Leave a comment saying why we need to build GDB. Hopefully one day we won't need to do this.

Reply:
Done.